Module-2 | Day-3/Promise.js

```diff
@@ -0,0 +1,75 @@
+var fs = require('fs');
+
+var readFile1 = function(){
+		
+	return new Promise((success,error) =>{
+
+    fs.readFile('file1.txt', function(err, data) {
+      if(err) 
+        return error(err);
+      else
+        return success(data);
+
```

Comment:
this callback could have been written in one line as `(err, data) => err ? error(err) : success(data)`

Reply:
Read about ternary operator in javascript.